e2e/test/BulkOperationsE2ETests.cs

```diff
@@ -12,7 +12,7 @@ namespace Microsoft.Azure.Devices.E2ETests
     [TestClass]
     [TestCategory("E2E")]
     [TestCategory("IoTHub")]
-    public class BulkOperationsE2ETests : IDisposable
+    public class BulkOperationsE2ETests : E2ETestBase, IDisposable
```

Comment:
Currently only two classes are on-boarded to this. I have added a task to onboard every other class.